examples/Python/MR/interactive/coil_combination.py

```diff
@@ -13,135 +13,137 @@
 # Author: Christoph Kolbitsch
 #
 
-## CCP PETMR Synergistic Image Reconstruction Framework (SIRF).
-## Copyright 2015 - 2017 Rutherford Appleton Laboratory STFC.
-## Copyright 2015 - 2017 University College London.
-## Copyright 2015 - 2017 Physikalisch-Technische Bundesanstalt.
-##
-## This is software developed for the Collaborative Computational
-## Project in Positron Emission Tomography and Magnetic Resonance imaging
-## (http://www.ccppetmr.ac.uk/).
-##
-## Licensed under the Apache License, Version 2.0 (the "License");
-##   you may not use this file except in compliance with the License.
-##   You may obtain a copy of the License at
-##       http://www.apache.org/licenses/LICENSE-2.0
-##   Unless required by applicable law or agreed to in writing, software
-##   distributed under the License is distributed on an "AS IS" BASIS,
-##   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-##   See the License for the specific language governing permissions and
-##   limitations under the License.
+# # CCP PETMR Synergistic Image Reconstruction Framework (SIRF).
+# # Copyright 2015 - 2017 Rutherford Appleton Laboratory STFC.
+# # Copyright 2015 - 2017 University College London.
+# # Copyright 2015 - 2017 Physikalisch-Technische Bundesanstalt.
+# #
+# # This is software developed for the Collaborative Computational
+# # Project in Positron Emission Tomography and Magnetic Resonance imaging
+# # (http://www.ccppetmr.ac.uk/).
+# #
+# # Licensed under the Apache License, Version 2.0 (the "License");
+# #   you may not use this file except in compliance with the License.
+# #   You may obtain a copy of the License at
+# #       http://www.apache.org/licenses/LICENSE-2.0
+# #   Unless required by applicable law or agreed to in writing, software
+# #   distributed under the License is distributed on an "AS IS" BASIS,
+# #   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# #   See the License for the specific language governing permissions and
+# #   limitations under the License.
 
 __version__ = '0.1.0'
 
 # import engine module
 import pGadgetron as pMR
 
 # import further modules
-import os, numpy
+import os
+import numpy
 import matplotlib.pyplot as plt
 
-#%% GO TO MR FOLDER
+# %% GO TO MR FOLDER
```

Comment:
Visually, it doesn't. However, when following a style guide with automatic warnings (as e.g. with Spyder and PEP8), it is much easier to fully the guide completely rather than cherry picking which of these we prefer.

Reply:
`%%` is there for a spyder cell. It seems that both `#%%` and `# %%` are [supported](https://docs.spyder-ide.org/editor.html).

Reply:
Yes exactly, forgot to mention that. `# %%` is an alternative to `#%%` which is PEP8 compatible but also generates a Spyder cell.

Reply:
Ah, well all good then :)